translations/deckbuilding.tex/pl.po

```diff
@@ -88,6 +93,29 @@ msgid ""
 "}\n"
 "\n"
 msgstr ""
+"\\subsection*{Ogólne Zasady Kart}\n"
+"\\begin{enumerate}\n"
+"  \\item Karty możesz zagrywać \\textbf{wyłącznie podczas twojej Tury} lub w \\hyperlink{Combat}{Walce}, w której walczy twój \\textbf{Bohater Główny}.\n"
+"  \\item Po zagraniu Karty, odrzuć ją na swój własny Stos Kart Odrzuconych\\index{Stos Kart Odrzuconych}.\n"
+"  \\item Jeśli musisz dobrać Kartę, a twoja Talia jest pusta, \\textbf{potasuj swój Stos Kart Odrzuconych}, tworząc nową Talię, z której będziesz dobierać Karty.\n"
+"  \\item Gdy twój Bohater zdobywa Kartę, weź ją \\textbf{bezpośrednio do ręki}, jeśli nie napisano inaczej.\n"
+"  \\item \\textbf{Przeszukując} (X)\\index{Przeszukaj (X)} Talię Umiejętności, Artefaktów lub Zaklęć, możesz spojrzeć na wierzchnie (X) Kart danej Talii, wziąć jedną z nich na rękę i odrzucić pozostałe \\textbf{ALBO} zamiast podglądać (X) Kart, dobrać wierzchnią Kartę ze Stosu Kart Odrzuconych tej Talii.\n"
+"  \\item Talie Umiejętności, Artefaktów i Zaklęć mają własne Stosy Kart Odrzuconych zbudowane podczas przygotowania  - pomagają one w rozpoznaniu tych Talii.\n"
+"    Jeśli w Talii skończą się Karty, potasuj ją i odrzuć wierzchnią Kartę tworząc nowy Stos Kart Odrzuconych.\n"
+"    Gdy Stos Kart Odrzuconych jest pusty, \\textbf{odnów go} wierzchnią Kartą tej Talii.\n"
+"  \\item Karty mają następujące typy efektów:\n"
+"  \\begin{itemize}\n"
+"    \\item \\textbf{Natychmiastowy} \\includesvg[height=10px]{\\svgs/instant.svg} Są wykonywane natychmiatowo.\n"
+"    \\item \\textbf{Aktywacja} \\includesvg[height=10px]{\\svgs/activation.svg} Muszą być zagrane podczas Aktywacji twojej Jednostki w Walce.\n"
+"    \\item \\textbf{Mapa} \\includesvg[height=10px]{\\svgs/map.svg} Nie mogą być zagrane podczas Walki.\n"
+"    \\item \\textbf{Krótkotrwały} \\includesvg[height=10px]{\\svgs/ongoing.svg} Trwają dopóki zostaną wykorzystane lub dopóki gracz, który je zagrał rozpocznie swoją kolejną Turę (którekolwiek nastąpi pierwsze).\n"
+"    \\item \\textbf{Długotrwały} \\includesvg[height=10px]{\\svgs/permanent.svg} \\index{Karty z efektem Długotrwałym} pozostają w grze dopóki nie zostaną odrzucone albo zamienione.\n"
+"      \\textbf{Możesz mieć tylko jedną Kartę z efektem Długotrwałym}; zagranie innej powoduje odrzucenie pierwszej.\n"
```

Comment:
Tu przydałby się przykład odrzucania takich kart. Ja osobiście nie jestem pewien co się dzieje gdy mam Balistę i kupie Namiot Medyka. Gdzie odrzucać Balistę? Zakrywam ją? Całkowicie tracę? To by się przydało dodać nawet w oryginale doprecyzować.

Reply:
Właśnie do takiego celu są Issues - możesz taki stworzyć i zostanie podjęta dyskusja na ten temat

Reply:
Nie mam za dużego doświadczenia w GitHubie. Ale jeżeli ktoś zgłosi go zamiast mnie, to byłoby super.